DRAFT: Add major changes to the constitution. (#44)
csesoc/csesoc-constitution: CONSTITUTION.rst

those notes shall be removed.

Preliminaries
=============

The Society
-----------

    CSESoc Constitution, Section 1

1. The official name of the society shall be Computer Science and Engineering Society of UNSW.

   1. The acronym of the society shall be CSESoc.

2. The society shall be affiliated to Arc @ UNSW Limited.
3. The aims and objectives of the society are:

   1. To fulfil the specific social needs of CSE students; 
   #. To promote computing through technical and nontechnical oriented events to all students of UNSW; and 
   #. To establish rapport with external organisations/groups that are enthusiastic about computing.

4. During the course of affiliation, the society shall comply with any requirements of Arc.
5. The society will be a Constituent society since it has an agreed association with School of 
   Computer Science and Engineering, which is ratified by Arc. 

Definitions
-----------

    CSESoc Consitution, Section 1
    Model Constitution, Section 1

1. In this constitution:

   #. *secretary* means:

      #. the person holding office under this constitution as secretary of CSESoc, or

>       #. if no person holds that office - the public officer of CSESoc.

Reviewer: I would suggest putting somewhere in the constitution who is/can be the public officer.

   #. *extraordinary general meeting* means a general meeting of CSESoc other than an annual general meeting.
   #. *the Act* means the Associations Incorporation Act 2009.
   #. *the Regulation* means the Associations Incorporation Regulation 2016.

#. *The University* shall mean the University of New South Wales; 
#. *Arc* shall mean Arc @ UNSW Limited; 
#. *Tharunka* and *Blitz* shall mean the official publications of Arc; 
#. *Re-affiliation* shall mean re-affiliation with Arc; 
#. *Full members* shall mean full members of the Computer Science and Engineering Society of UNSW; 
#. *Associate members* shall mean associate members of the Computer Science and Engineering Society of UNSW; 
#. *The Executive* shall mean the Executive of the Computer Science and Engineering Society of UNSW; 
#. *A Portfolio* shall mean a group of members of the society, selected by The Executive to perform a particular task or tasks, as delegated to by The Executive.
#. *The Annual General Meeting* shall mean the Annual General Meeting of the CSE Society; 
#. An *academic day* shall mean a day during the first, second or third term of the University’s 

>    cademic year which is not a Saturday, Sunday, Public Holiday or University Holiday; 

Reviewer: ```suggestion
  academic year which is not a Saturday, Sunday, Public Holiday or University Holiday;
```

#. A *business* day shall mean a day which is not a Saturday, Sunday, Public Holiday, 
   niversity Holiday or within a University exam period; 
#. The *School of CSE* or any references to the acronym *CSE* shall mean Computer 

>    cience and Engineering; and 

Reviewer: ```suggestion
   Science and Engineering; and 
```

#. *Subjects* shall mean units of study offered by the University in progression to the 

>    ward of a degree. 

Reviewer: This line appears to be unused.

      #. That person is currently listed as enrolled in:

         #. A Computer Science specialisation; or
         #. A Computer Engineering, Software Engineering or Bioinformatics Engineering specialization; or
         #. A postgraduate program administered by CSE (research or coursework).
      #. Full membership shall not require any fee.
      #. Full membership shall cease when a person is no longer enrolled in one of the courses described in the previous sub-section.

#. Associate membership shall be subject to the following conditions:

   #. A person shall be an associate member of the society if:

      #. They are currently enrolled in a course administered by the School of Computer Science and Engineering.
   #. An associate member whose membership is by virtue of enrollment in a course shall cease to be a member of CSESoc at the beginning of the term after the term of the course they enrolled in, unless they would otherwise retain that membership.
   #. The executive shall have the power to reasonably interpret which term constitutes the "term after the term of the course they enrolled in".
   #. A person shall also be an associate member of the society if:

>       #. They have applied to a member of the club Executive of CSESoc to become an associate member, and

Reviewer: Minor thing, but would recommend removing references to "the club" for consistency's sake.

      #. They would not cause a violation of the affiliation requirements of Arc, and
      #. They pay a membership fee, as set by the club Executive, and
      #. They complete a membership form prepared by the club Executive.
   #. An associate member whose membership is by application to the Executive shall cease to be a member of CSESoc at the end of the calendar year, unless they would otherwise retain that membership.
#. Notwithstanding other provisions of this section, a person ceases to be a member of CSESoc if the person:

   #. dies, or
   #. notifies the Executive in writing to request their membership be terminated, or

>    #. is removed from the Executive under the provisions described in `Disciplining of Members`_.

Reviewer 2: Dead link - should this link to Impeachment instead?

Author: I think a few of the links are broken. I'll check them out, thanks :+1:

Membership entitlements not transferable
----------------------------------------

    This section has been taken from Section 5 of the Model Consitution.

#. A right, privilege or obligation which a person has by reason of being a member of CSESoc:

   #. is not capable of being transferred or transmitted to another person, and
   #. terminates on cessation of the person’s membership.

Register of Members
-------------------

> #. The secretary must establish and maintain a register of members of CSESoc (whether in written or electronic form) specifying the name and postal, residential or email address of each person who is a member of CSESoc together with the date on which the person became a member.

Reviewer: Nothing that needs to be changed _per se_, but be aware that the law requires extra records to be kept that aren't listed in the Model Constitution. The following records must be kept about the Executive:
* name, date of birth, residential address
* dates they enter/leave the Executive
* names and start/end dates of anyone in the positions of "president, vice-president (N/A), secretary or treasurer"

Resolution of disputes
----------------------

    DISCUSS WITH ARC LEGAL -> DO I NEED TO SAY 1 or 4 or 5

#. The society shall publicly maintain a Grievance Resolution Policy and Procedure.

   #. This Policy and Procedure shall initially be the Grievance Resolution Policy and Procedure of the Unincorporated Association which incorporates under this constitution.
   #. This Policy and Procedure may be modified by a vote of the Executive.

#. A dispute between a member and another member (in their capacity as members) of the association shall be referred to the CSESoc Grievance officer; who shall act in accordance with the Grievance Resolution Policy and Procedure.
#. A dispute between a member or members and CSESoc, is to be referred to a Community Justice Centre for mediation under the Community Justice Centres Act 1983.
#. If a dispute is not resolved by mediation within 3 months of the referral to a Community Justice Centre, the dispute is to be referred to arbitration.

> #. The Commercial Arbitration Act 2010 applies to a dispute referred to arbitration.

Reviewer: This is just the default given in the Model Constitution. I'm pretty sure you can use a different dispute resolution policy if you want (i.e. you don't have to use their exact procedure).

   #. Cease to be a full member of the society; or 
   #. Are removed from the Executive position after the following procedure:

      #. A motion is carried by a two thirds majority vote of the Executive that are not subject to vacancy proceedings to instigate impeachment proceedings; or
      #. The Executive is petitioned by fifteen (15) members to instigate impeachment proceedings;
      #. The members of the society are formally notified of the motion upon notice of an Extraordinary General Meeting under Section 4.2;
      #. The Executive  member concerned is notified of the motion and reasons for termination in writing at least five (5) academic days prior to the Extraordinary General Meeting;
      #. The Executive member concerned is given five (5) minutes to speak against the motion at the Extraordinary General Meeting; and
      #. The motion is carried by the Extraordinary General Meeting.

   #. Any vacancy in the society Executive must be filled by following the election procedure under Section 8.

      #. If there is a tie for the vacant executive position, the current executive shall have a casting vote in the election.

>       #. The candidate must accept the offer of Executive position at that meeting, or alternatively at an Extraordinary General Meeting before they may fill the position.

Reviewer: This whole vacancy bit appears to be somewhat duplicated in the "Casual Vacancies" section way down below.

#. At the Handover Meeting, the Executive shall pass on all knowledge of, and advice regarding, the Society to the Executive-elect. 
#. At the conclusion of the Handover Meeting, the Executive is dissolved and the Executive- elect assume their elected positions. 
 

General Meetings
================

General meetings - requirements
-------------------------------

    Taken from section 5 of the CSESoc Constitution.
    Proxies were discussed in section 35 of the Model Constitution,
    and we must allow/disallow them as a requirement of the act.
    The CSESoc Constitution allows them, so we do too.

#. General meeting requirements for all general meetings are as follows:

   #. All voting at general meetings shall be with a simple majority required for a resolution to be passed;
   #. Except where specified otherwise, each member (full, or associate) shall be allowed 1 vote.
   #. Proxies shall be allowed in meetings and the procedure shall comply with the requirements of Arc;
   #. In the case of equality of voting, the motion shall be defeated. 
   #. Constitutional changes must be in the form of a motion on notice to an Annual or Extraordinary General Meeting; and 

>    #. Constitutional changes passed at an Annual or Extraordinary General Meeting will not take effect until ratified by Arc. 

Reviewer: ```suggestion
   #. Constitutional changes passed at an Annual or Extraordinary General Meeting must be approved by Arc for CSESoc to remain affiliated with Arc.
```
This should be updated to the current Arc model constitution wording. As CSESoc would now have its own legal identity, Arc cannot technically prevent constitution changes from taking effect. (Arc affiliation would be suspended though.)

   #. to elect members of the Executive;
   #. to receive and consider any financial statement or report required to be submitted to members under the Act.

    This requirement will change in another commit to be made.
      
   #. Nominations for the Executive elections shall open, pursuant to `Elections`.

   #. Full minutes of this meeting, including a list of the new Executive, written financial reports, and constitutional amendments, shall be forwarded to Arc within ten (10) business days of the Handover Meeting; and

#.  Each full member is entitled to one vote at an annual general meeting.

   #. Should the society have less than 15 full members, half of the number of full members shall constitute a quorum.

#. An annual general meeting must be specified as that type of meeting in the notice convening it.

Extraordinary general meetings - calling of
-------------------------------------

    Copied from section 27 of the Model Constitution, but with extra stuff from Section 5 of the CSESoc constitution.

#. The Executive may, whenever it thinks fit, convene a special general meeting of CSESoc.
#. Extraordinary General Meetings must be held on an academic day.

> #. Each full member is entitled to one vote.

Reviewer: Seems to be redundant with the "Voting" section.

   #. Such a requisitioned meeting must be held within twenty-one (21) academic days, but no 
              sooner than five (5) academic days.
   #. If the Executive fails to convene a special general meeting to be held within 1 month after the date on which a requisition of members for the meeting is lodged with the secretary, any one or more of the members who made the requisition may convene a special general meeting to be held not later than 3 months after that date.

   #. A special general meeting convened by a member or members as referred to in this clause must be convened as nearly as is practicable in the same manner as general meetings are convened by the Executive.

#. For the purposes of subclause (3):

   #. a requisition may be in electronic form, and
   #. a signature may be transmitted, and a requisition may be lodged, by electronic means.

#. The format, procedures, notice and quorum for an Extraordinary General Meeting shall be the same as for an Annual General Meeting, except that the Executive nominations shall not take place unless specifically notified.

Notice
------

    Copied from section 28 of the Model Constitution.

> #. Except if the nature of the business proposed to be dealt with at a general meeting requires a special resolution of CSESoc, the secretary must, at least 14 days before the date fixed for the holding of the general meeting, give a notice to each member specifying the place, date and time of the meeting and the nature of the business proposed to be transacted at the meeting.

Reviewer: This section says 14 days' notice is required, but in the _"Annual general meetings - holding of"_ section, it says that only 5 academic days is required.

Quorum for general meetings
---------------

    Copied from section 29 of the Model Constitution. We require 15 members because Arc.

#. No item of business is to be transacted at a general meeting unless a quorum of members entitled under this constitution to vote is present during the time the meeting is considering that item.
#. Fifteen members present (being members entitled under this constitution to vote at a general meeting) constitute a quorum for the transaction of the business of a general meeting.

   #. Should the society have less than 15 full members, half of the number of full members shall constitute a quorum.

#. If within half an hour after the appointed time for the commencement of a general meeting a quorum is not present, the meeting:

   #. if convened on the requisition of members—is to be dissolved, and
   #. in any other case—is to stand adjourned to the same day in the following week at the same time and (unless another place is specified at the time of the adjournment by the person presiding at the meeting or communicated by written notice to members given before the day to which the meeting is adjourned) at the same place.

> #. If at the adjourned meeting a quorum is not present within half an hour after the time appointed for the commencement of the meeting, the members present (being at least 3) are to constitute a quorum. 

Reviewer 3: +1 - from memory a  quorum of 15 members or half the society, whatever is lesser

---------------

    Copied from section 41 of the Model Constitution.

#. Subject to the Act and the Regulation, CSESoc must apply its funds and assets solely in pursuance of the objects of CSESoc and must not conduct its affairs so as to provide a pecuniary gain for any of its members.

Disolution of the society
-------------------------

    This is copied from section 7 of the CSESoc constitution.

#.  Dissolution of the Computer Science and Engineering Society of UNSW will occur after the following conditions have been met:

   #. An Extraordinary General Meeting is petitioned in writing as set out in 4.11; 
   #. Procedures for notification as set out in 4.1.3 are followed, and the reasons for the proposed dissolution are included with the notification to Arc; 
   #. Quorum for the meeting to dissolve the society shall be twenty (20) members or three-quarters of the society membership, whichever is the lesser; 
   #. No other business may be conducted at the meeting to dissolve the society; 
   #. After the petitioning body has stated its case any opposition must be given the opportunity to reply, with at least ten minutes set aside for this purpose; 

>    #. A vote is taken and the motion to dissolve lapses if opposed by fifteen (15) or more members of the society; and 

Reviewer: Note that the dissolution motion must be passed by special resolution, i.e. 21 days' notice and 75% vote is required.

#. Arc must give twenty (20) academic days notice in Tharunka and/or Blitz and/or the Arc website, and in writing to the last known Co-Presidents before dissolving the society in this way. 
#. On dissolution of the society, the society is not to distribute assets to members. All assets are to be distributed to an organisation with similar goals or objectives that also prohibits the distribution of assets to members. This organisation may be nominated at the dissolution meeting of the society. If no other legitimate organisation is nominated, Arc will begin procedures to recover any property, monies or records belonging to the society which it perceives would be useful to other Arc-affiliated clubs. 
        
Distribution of property on winding up of association 
---------------

    Copied from section 42 of the Model Constitution.
    Note from the model constituion:
    Section 65 of the Act provides for distribution of surplus property on the winding up of an association.

> #. Subject to the Act and the Regulations, in a winding up of CSESoc, any surplus property of CSESoc is to be transferred to another organisation affiliated with Arc, with similar objects and which is not carried on for the profit or gain of its individual members.

Reviewer: This clause and the one above should be merged together; seems redundant.

   #. in the case of a notice given or served personally, on the date on which it is received by the addressee, and
   #. in the case of a notice sent by pre-paid post, on the date when it would have been delivered in the ordinary course of post, and
   #. in the case of a notice sent by facsimile transmission or some other form of electronic transmission, on the date it was sent or, if the machine from which the transmission was sent produces a report indicating that the notice was sent on a later date, on that date.

#. Notices made to the entire membership of CSESoc must be emailed to those members, in addition to any other form those notices take.

Financial year
---------------

    Notes from the Model Constitution:
    1. Schedule 1 to the Act provides that an association’s constitution is to address CSESoc’s financial year.
    2. Clause 19 of the Regulation contains a substitute clause 47 for certain associations incorporated under CSESocs Incorporation Act 1984.
	
#. The financial year of CSESoc is:

   #. the period of time commencing on the date of incorporation of CSESoc and ending on the following 31 December, and

>    #. each period of 12 months after the expiration of the previous financial year of CSESoc, commencing on 1 July and ending on the following 30 June.

Reviewer: This line and the one immediately above are in conflict. The ending date needs to be the same in both clauses.

   university Holiday or within a University exam period; 
#. The *School of CSE* or any references to the acronym *CSE* shall mean Computer 
   Science and Engineering; and 
#. *Subjects* shall mean units of study offered by the University in progression to the 
   award of a degree. 
#. In this constitution:

   #. a reference to a function includes a reference to a power, authority and duty, and
   #. a reference to the exercise of a function includes, if the function is a duty, a reference to the performance of the duty.

#. The provisions of the Interpretation Act 1987 apply to and in respect of this constitution in the same manner as those provisions would so apply if this constitution were an instrument made under the Act.

Membership
==========

    Part 2 of this Constitution is structured around Part 2 of the Model Constitution, with sections merged
    or added from the CSESoc Constitution.

Membership Generally
--------------------

    This section has been entirely taken from sections 2.1 to 2.5 of the CSESoc Constitution.
    It has been reformatted, but no substantive additions have been made.

> #. There shall be two types of membership: *full member*, and *associated member*.

Reviewer: ```suggestion
#. There shall be two types of membership: *full member*, and *associate member*.
```

   #. One (1) Treasurer;
   #. One (1) Secretary;
   #. One (1) Arc Delegate; and
   #. One (1) Grievance Officer.

#. A member is permitted to hold no more than one (1) Executive position, unless one of the positions is of the Grievance Officer, in which case a member can hold two (2) Executive positions.
#. Subject to the Act, the Regulation, this constitution and any resolution passed by CSESoc in general meeting, the Executive shall:

   #. Ensure that through the activities of the Society’s working groups, its aims are being fulfilled;
   #. Maintain the finances of the society;
   #. Communicate with the membership of the Society; and
   #. Maintain and review policies & procedures of the Society, including its Grievance Resolution Policy & Procedure.
   #. Control and manage the affairs of CSESoc, and
   #. Exercise all the functions that may be exercised by CSESoc, other than those functions that are required by this constitution to be exercised by a general meeting of members of CSESoc, and
   #. Have power to perform all the acts and do all things that appear to the Executive to be necessary or desirable for the proper management of the affairs of CSESoc.

> #. There is no maximum number of consecutive terms for which a Executive member may hold office. 

Reviewer: Unless I've missed it, the Act requires the constitution to mention how long "the terms of office of the committee members" is. You can just copy/paste in the model constitution wording here:

> Each member of the Executive is, subject to this constitution, to hold office until immediately before the election of Executive members at the annual general meeting next following the date of the member’s election, and is eligible for re-election.

Author: For this, we specify:
> #. There is no maximum number of consecutive terms for which a Executive member may hold office. 
We also specify that executives are dissolved by handover meeting.
I've added a note that executive members hold office until the executive is dissolved by handover meeting (subject to the other provisions).

Reviewer: Ah right, I've missed that indeed. Thanks :+1:

#. For the purpose of this constitution, a notice is taken, unless the contrary is proved, to have been given or served:

   #. in the case of a notice given or served personally, on the date on which it is received by the addressee, and
   #. in the case of a notice sent by pre-paid post, on the date when it would have been delivered in the ordinary course of post, and
   #. in the case of a notice sent by facsimile transmission or some other form of electronic transmission, on the date it was sent or, if the machine from which the transmission was sent produces a report indicating that the notice was sent on a later date, on that date.

#. Notices made to the entire membership of CSESoc must be emailed to those members, in addition to any other form those notices take.

Financial year
---------------

    Notes from the Model Constitution:
    1. Schedule 1 to the Act provides that an association’s constitution is to address CSESoc’s financial year.
    2. Clause 19 of the Regulation contains a substitute clause 47 for certain associations incorporated under CSESocs Incorporation Act 1984.
	
#. The financial year of CSESoc is:

>    #. the period of time commencing on the date of incorporation of CSESoc and ending on the following 31 December, and
>    #. each period of 12 months after the expiration of the previous financial year of CSESoc, commencing on 1 January and ending on the following 31 December.

Reviewer: ```suggestion
   #. the period of time commencing on the date of incorporation of CSESoc and ending on the following 30 June, and
   #. each period of 12 months after the expiration of the previous financial year of CSESoc, commencing on 1 July and ending on the following 30 June.
```

The AGM must be held within 6 months of the financial year end. Given that Arc's required AGM period is usually in Term 3, I would recommend using 1 July as the financial year instead.
